examples/asr/avsr_rnnt/models/resnet.py

```diff
@@ -0,0 +1,251 @@
+import math
+import torch.nn as nn
+import pdb
+import torch
+
+class Swish(nn.Module):
```

Comment:
pytorch already has this ([torch.nn.SiLU](https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html))